samples/getting-started/simulation/LargeSimulation.ipynb

```diff
@@ -0,0 +1,2459 @@
+    "\n",
+    "For more information about the sparse simulator, please see [Jaques and Häner (arXiv:2105.01533)](https://arxiv.org/abs/2105.01533).\n",
+    "\n",
+    "In this sample, we'll look at the advantages of the sparse simulator over the full-state simulator.\n",
+    "\n",
+    "Below, we'll create a few clusters (groups) of qubits, each prepared in a [GHZ state](https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "prime-horror",
+   "metadata": {},
+   "source": [
+    "To run the cells below, click the cell and press \\<Ctrl+Enter>. The same shortcut will let you exit from the Edit mode of a Markdown cell like the one you are reading. To see the other shortcuts press \\<Esc>, \\<h>. Press \\<Ctrl+Shift+p> to see the command pallete. \\<Esc> to exit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "shaped-office",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[]",
+      "text/html": [
+       "<ul></ul>"
+      ],
+      "text/plain": []
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "open Microsoft.Quantum.Arrays;        // Contains functions for creating and manipulating arrays of data.\n",
+    "open Microsoft.Quantum.Diagnostics;   // Contains subroutines that help ensure correctness of Q# programs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "affiliated-value",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[\"PrepareAndMeasureGhzState\"]",
+      "text/html": [
+       "<ul><li>PrepareAndMeasureGhzState</li></ul>"
+      ],
+      "text/plain": [
+       "PrepareAndMeasureGhzState"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "/// # Summary\n",
+    "/// Prepares, measures, and resets the GHZ states.\n",
+    "///\n",
+    "/// # Input\n",
+    "/// ## nQubits\n",
+    "/// Number of qubits to allocate (must be multiple of the nClusters input).\n",
+    "/// \n",
+    "/// ## nClusters\n",
+    "/// Number of clusters to break-up qubits into.\n",
+    "operation PrepareAndMeasureGhzState(nQubits : Int, nClusters : Int) : Unit {\n",
+    "    Message(\"Let's run a big simulation!\");\n",
+    "\n",
+    "    Message($\"Allocating {nQubits} qubits...\");\n",
+    "    use qs = Qubit[nQubits];                        // Allocate the qubits.\n",
+    "    Message(\"Finished allocating!\");\n",
+    "\n",
+    "    let clusterSize = nQubits / nClusters;          // Number of qubits in every cluster.\n",
```

Comment:
How readable are these comments on narrower screens? It's also worth keeping in mind that in the Azure portal, other blades can claim a fair bit of horizontal space. It may be worth moving these comments to be above relevant lines instead of to the side.